src/Services/FHIR/FhirLocationService.php

```diff
@@ -60,43 +66,44 @@ public function parseOpenEMRRecord($dataRecord = array(), $encode = false)
 
         $locationResource->setStatus("active");
 
-        $locationResource->setAddress(array(
-            'line' => [$dataRecord['street']],
-            'city' => $dataRecord['city'],
-            'state' => $dataRecord['state'],
-            'postalCode' => $dataRecord['postal_code'],
-        ));
-
+        // TODO: what is this `s Home thing, seems really wierd... and its not documented
         if (!empty($dataRecord['name'] && $dataRecord['name'] != "`s Home")) {
```

Comment:
Agree this does not make sense and don't think it's stored in the database like this, so unsure where this is coming from.

Reply:
So I found the original code in LocationService was adding the person's name and Home.  I've just set it to be Home Address